src/bb-modules/Formbuilder/Service.php

```diff
@@ -344,12 +344,12 @@ public function formFieldNameExists($data)
         $sql = ("
         SELECT COUNT( * )
         FROM  `form_field`
-        WHERE form_id = :form_id
-        AND name =  :field_name
-        AND id != :field_id
+        WHERE form_id = form_id
```

Comment:
Please explain :form_id. Why did you not use just form_id here?

Reply:
the use of ":" means the field is a parameter. The previous syntax with :form_id, :field_name and :field_id are the correct ones.

https://www.php.net/manual/en/pdostatement.bindparam.php